syslibs/comm/TCPbind/source/aresWorker.c

```diff
@@ -180,7 +180,7 @@ void* workerThread( void* lpParam ){
 							KS_logfile_debug(("aresWorkerThread: address not usable"));
 							continue;
 						}
-						KS_logfile_debug(("file %s\nline %u:\tconnectiong to %i", __FILE__, __LINE__, sockfd));
+						KS_logfile_debug(("file %s\nline %u:\tconnectiong", __FILE__, __LINE__));

```

Comment:
Ups :-) did’t see that

Problem was the sockfd: copy paste from another function

 

Von: Sten Grüner [mailto:notifications@github.com] 
Gesendet: Dienstag, 24. Mai 2016 10:20
An: acplt/rte
Cc: Evertz, Lars; State change
Betreff: Re: [acplt/rte] minor improvements / fixes (ksxdr and TCPbind) (#17)

 

In syslibs/comm/TCPbind/source/aresWorker.c <https://github.com/acplt/rte/pull/17#discussion_r64346866> :

> @@ -180,7 +180,7 @@ void* workerThread( void* lpParam ){
>                                                    KS_logfile_debug(("aresWorkerThread: address not usable"));
>                                                    continue;
>                                             }
> -                                           KS_logfile_debug(("file %s\nline %u:\tconnectiong to %i", __FILE__, __LINE__, sockfd));
> +                                           KS_logfile_debug(("file %s\nline %u:\tconnectiong", __FILE__, __LINE__));

connectiong?

—
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub <https://github.com/acplt/rte/pull/17/files/d0f3b33a2b85e8f8057c3eabae304f4bcdbab144#r64346866>   <https://github.com/notifications/beacon/AIyEwt0OcG5FSzuTvBNVpFV69vBPMX98ks5qErSSgaJpZM4IlMvt.gif>